Automattic/wp-calypso
# VideoPress HQ: Remove Intro content and Options step

Related to #

The entire flow right now should be an intro (never seen, just a landing spot to determine where to navigate to next), a signup step (if not currently logged in) and a processing step.

**_Processing step_**
<img width="1024" alt="image" src="https://github.com/Automattic/wp-calypso/assets/4081020/df274af7-e234-497f-921e-1a1e4c4d0c39">

**_Already logged in state_**
<img width="300" alt="image" src="https://github.com/Automattic/wp-calypso/assets/4081020/a4d933d3-9b86-4cf1-ae4a-957f5175f0e4">

## Proposed Changes

* remove the Options step from onboarding, send user directly to site creation and then the site (note: this will be preempted by a "you already have a trial" check in a future PR)
* fix bug with `processing` step for VideoPress not liking being loaded immediately after new login

## Testing Instructions



* `yarn && yarn start` or use calypso live link below
* in an incognito browser, visit `/setup/videopress-tv/processing`
* you should be redirected to `/intro` page which itself redirects to sign up
* move through the flow, sign in with a new or existing account
* after signin you should see the Processing page and then be redirected to your new site. The site title should be `VideoPress TV`
* visit the `/setup/videopress-tv/intro` page again which should send you to the account screen to verify you are using the account you want the site on. Click continue and you'll be taken to the processing step and a new site should be created (simulating the "already logged in" experience)
* regression test another flow to ensure the Processing step messages continue to work there (ex /setup/link-in-bio)
 
## Pre-merge Checklist
